### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by fouadkachlan**

### CODE REVIEW

File: App.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.todocontainer {
+    display: flex;
```
Comment (jihad292) [inline App.css]: inline style no need for this file

File: Button/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react"
+
+
+const Button = ({onClick, btnStyle, children}) => {
+    return (
+        <button style={btnStyle} onClick={onClick}>
```
Comment (jihad292) [inline Button/Button.jsx]: buttonStyle

File: TodoList/TodoList.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  const deleteTodo = (taskToDelete) => {
+    const updatedTodos = todos.filter((todo) => todo !== taskToDelete);
+    setTodos(updatedTodos);
+  };
+  
+  return (
+    <div className="todocontainer">
+      <h2>To-Do List</h2>
+      <input
```
Comment (jihad292) [inline TodoList/TodoList.jsx:44]: custom component for html

File: TodoList/TodoList.jsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                          backgroundCcolor:"chartreuse"}}
+       >
+        Add Task
+        </Button>
+    </div>
+  );
+};
+
+export default TodoList;
+//Questions answers
```
Comment (jihad292) [inline TodoList/TodoList.jsx]: by email

File: App.css (hunk 1 above)
Comment (fouadkachlan) [reply App.css]: solved

File: Button/Button.jsx (hunk 2 above)
Comment (fouadkachlan) [reply Button/Button.jsx]: solved

File: App.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    justify-content: center;
+    flex-direction: column;
+    background-color:bisque;
+    margin: 5rem;
+    padding:50px 50px 50px 50px;
```
Comment (jihad292) [inline App.css]: delete this file it is not needed

File: Button/Button.jsx (hunk 2 above)
Comment (jihad292) [reply Button/Button.jsx]: still btnStyle ??

File: TodoList/TodoList.jsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+//if we have an array const numbers=[100,200,300];
```
Comment (jihad292) [inline TodoList/TodoList.jsx]: delete it here

**Comment by jihad292** [comment]
Fouad Please focus on the comments and read them carefully

File: Button/Button.jsx (hunk 2 above)
Comment (fouadkachlan) [reply Button/Button.jsx]: solved I updated In another way

File: TodoList/TodoList.jsx (hunk 3 above)
Comment (fouadkachlan) [reply TodoList/TodoList.jsx:44]: Done, I created a new folder called CustomDiv, and inside it i place my custom component called CustomDiv.jsx, Please check them out

File: TodoList/TodoList.jsx (hunk 4 above)
Comment (fouadkachlan) [reply TodoList/TodoList.jsx]: Done

File: App.css (hunk 5 above)
Comment (fouadkachlan) [reply App.css]: Done

File: TodoList/TodoList.jsx (hunk 6 above)
Comment (fouadkachlan) [reply TodoList/TodoList.jsx]: Done

**Comment by fouadkachlan** [comment]
> Fouad Please focus on the comments and read them carefully

Done

File: Button/Button.jsx [hunk 7]
```diff
@@ -0,0 +1,18 @@
+import React from "react"
+
+
+const Button = ({onClick,children}) => {
+    return (
+        <button style={{
+            padding:"15px",
+            border: "20px",
+            borderRadius: "50px 50px",
```
Comment (jihad292) [inline Button/Button.jsx]: suppose that these styles are default for every button . but is we want to add more on this styles by sending them as props. how would you do it?

File: App.css [hunk 8]
```diff
@@ -0,0 +1 @@
+
```
Comment (jihad292) [inline App.css]: delete the file not only the content

Comment (fouadkachlan) [reply App.css]: Done

File: Button/Button.jsx (hunk 7 above)
Comment (fouadkachlan) [reply Button/Button.jsx]: Done

**Pull request closed without merging**